src/main/java/com/practice/PRACTICE/service/EmpServiceImpl.java

```diff
@@ -44,6 +45,22 @@ public Employee updateEmp(Employee employee, long id) throws ResourceNotFoundExc
         Employee updated = this.empRepo.save(employee);
         return updated;
     }
+    @Override
+    public Employee findByName(String fName, String lName) throws ObjectNotFound {
+        List<Employee> employees = this.getAllEmployee();
+        Employee res = null;
+        for (Employee e : employees) {
+            if (e.getLastName().equalsIgnoreCase(lName) && e.getFirstName().equalsIgnoreCase(fName)) {
+                res =   e;
+                break;
+            }
+
+        }
+        if (res == null) {
+            throw  new ObjectNotFound("OBJECT_NOT_FOUND");
```

Comment:
create a constant variable for this `OBJECT_NOT_FOUND` string. Better to create a `static class` inside `util` package that will have constants for everything which can be used throughout the project. below is an example

```
class Constants {
    private final String OBJECT_NOT_FOUND = "object not found" ;
    private final String INVALID_INPUT_PARAMETERS  = "input parameters are not valid in the request body";
    etc ...
}
```